cartographer_ros/scripts/remove_leading_slashes.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
`python3` should be possible, right?

Reply:
dunno. the ROS packages seem to be all python2 on my system.